src/Components/Inquiry/config.ts

```diff
@@ -31,14 +31,7 @@ export const useEngine = ({ context, onDone }: UseEngine) => {
       workflow: [
         {
           askSpecialist: {
-            true: [
-              "Specialist",
-              {
-                isLoggedOut: {
-                  true: ["Account"],
-                },
-              },
-            ],
+            true: ["Specialist"],
```

Comment:
suggestion (non-blocking): I understand the problem from the PR description, but I don't understand why this solves it. Perhaps a bit of context about why this solves the problem in the description would be helpful.

Reply:
Great point! just updated it :)